libraries/chain/chain_controller.cpp

```diff
@@ -586,6 +578,11 @@ static void validate_shard_locks(const vector<shard_lock>& locks, const string&
 
 void chain_controller::__apply_block(const signed_block& next_block)
 { try {
+   optional<fc::time_point> processing_deadline;
+   if (!_currently_replaying_blocks && _limits.max_push_block_us.count() > 0) {
+      processing_deadline = fc::time_point::now() + _limits.max_push_transaction_us;
```

Comment:
should this be:
+ _limits.max_push_block_us;

Reply:
yep, 11a6d1d fixes